frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/speedsearch/DiscoverAction.java

```diff
@@ -38,51 +39,69 @@ public IEnhancedLambdaLogger log() {
   @UIEffect
   public void actionPerformed(AnActionEvent anActionEvent) {
```

Comment:
This action is going to be rarely tested in everyday use... consider adding a test case for discover to `UITestSuite` (as a first thing the test would need to wipe out or delete machete file)